tests/reactive_fluid_transport.prm

```diff
@@ -0,0 +1,155 @@
+# This is a 1D test case for an insoluble fluid moving upwards in a
+# narrow channel through porous coupled two-phase flow.
+set Adiabatic surface temperature          = 1600
+set Nonlinear solver scheme                = iterated Advection and Stokes
+set Output directory                       = output
+set Max nonlinear iterations               = 10
+set Nonlinear solver tolerance             = 1e-5
+
+# The number of space dimensions you want to run this program in.
+set Dimension                              = 2
+
+set End time                               = 1e2
+
+
+set Use operator splitting                     = false
+
+
+# There are two compositional fields, one that tracks the amount of free fluid
+# (the porosity) and one that tracks the amount of bound fluid (bound_fluid).
+subsection Compositional fields
+  set Number of fields = 2
+  set Names of fields = porosity, bound_fluid
+end
+
+# Initially, there is no free water (the porosity is zero) and the water content
+# of the material is 1%.
+subsection Initial composition model
+  set Model name = function
+  subsection Function
+    set Function expression = if (y<=2e2,0.1,0); 0;
```

Comment:
Does this actually work with the semicolon at the end?
```suggestion
    set Function expression = if (y<=2e2,0.1,0); 0
```

Reply:
It does run, but good catch (it should be removed)